docs/manual/manual.adoc

```diff
@@ -580,30 +580,63 @@ command-line argument. You can use `--results-arf` to generate a Result DataStre
 $ oscap xccdf eval --benchmark-id benchmark_id --results-arf arf-results.xml scap-ds.xml
 --------------------------------------------------------------------------------------
 
-==== Result STIG Viewer
+==== Results for STIG Viewer
 
-If you want to import the XCCDF scan results to DISA STIG Viewer but your Rule IDs don't
-match the DISA's ones, you can use the `--stig-viewer` command-line argument along with
-a special reference in your Rules to generate XCCDF result files that can be imported by
-DISA STIG Viewer.
+DISA STIG Viewer is a graphical user interface (GUI) application that enables
+easy viewing of SCAP formatted Security Technical Implementation Guides
+(STIGs).  For more information on DISA STIG Viewer go to
+https://public.cyber.mil/stigs/srg-stig-tools/[SRG / STIG Tools].
```

Comment:
see the https://public.cyber.mil/stigs/srg-stig-tools/[SRG / STIG Tools] website.